arcade/gui/examples/scroll_area.py

```diff
@@ -22,12 +22,13 @@
 from arcade import Window
 from arcade.gui import UIManager, UIWidget, Property, Surface, UIDummy, UIEvent, bind, \
     UIMouseDragEvent, UIMouseScrollEvent, UIMouseEvent, UIBoxLayout, UIFlatButton, UIInputText
+from arcade.types import Point
 
 
 class UIScrollArea(UIWidget):
-    scroll_x = Property(default=0)
-    scroll_y = Property(default=0)
-    canvas_size = Property(default=(300, 300))
+    scroll_x = Property[float](default=0.0)
+    scroll_y = Property[float](default=0.0)
+    canvas_size = Property[Point](default=(300.0, 300.0))
```

Comment:
I think you can remove [float], and stuff. Unless there is something I am misunderstanding